lib/model/QubitInformationObject.php

```diff
@@ -2599,25 +2604,25 @@ public function setPhysicalObjectByName($physicalObjectName, $options)
     }
   }
 
-  public function importPhysicalObject($location, $name = false, $type = false, $label = false)
+  public function importPhysicalObject($location, $name, $options = array())
   {
-    if ($label && $type)
+    if ($options['label'] && $options['type'])
```

Comment:
I think the warning occurs when options is not initiated as a default array, otherwise in php `if (null)` is evaluated as false.